app/views/bill-runs/setup/region.njk

```diff
@@ -27,35 +26,33 @@
       }) }}
   {% endif %}
 
-  <div class="govuk-body">
-    <form method="post">
-      <input type="hidden" name="wrlsCrumb" value="{{wrlsCrumb}}"/>
+  <form method="post">
+    <input type="hidden" name="wrlsCrumb" value="{{wrlsCrumb}}"/>
 
-      {% set regionItems = [] %}
-      {% for region in regions %}
-        {% set regionItem = { text: region.displayName, value: region.id, checked: region.id === selectedRegion } %}
+    {% set regionItems = [] %}
+    {% for region in regions %}
+      {% set regionItem = { text: region.displayName, value: region.id, checked: region.id === selectedRegion } %}
 
-        {# Push our item into the region items array #}
-        {% set regionItems = (regionItems.push(regionItem), regionItems) %}
-      {% endfor %}
+      {# Push our item into the region items array #}
+      {% set regionItems = (regionItems.push(regionItem), regionItems) %}
+    {% endfor %}
 
-      {{ govukRadios({
-        attributes: {
-          'data-test': 'bill-run-region'
-        },
-        name: 'region',
-        errorMessage: error,
-        fieldset: {
-          legend: {
-            text: pageTitle,
-            isPageHeading: true,
-            classes: 'govuk-fieldset__legend--l govuk-!-margin-bottom-6'
-          }
-        },
-        items: regionItems
-      }) }}
+    {{ govukRadios({
+      attributes: {
+        'data-test': 'bill-run-region'
+      },
+      name: 'region',
+      errorMessage: error,
+      fieldset: {
+        legend: {
+          text: pageTitle,
+          isPageHeading: true,
+          classes: 'govuk-fieldset__legend--l govuk-!-margin-bottom-6'
```

Comment:
as you're setting the isPageHeading to true I don't think you need to set the legend to large `govuk-fieldset__legend--l` as that should be handled by the heading class that gets added already. would need to double check the differences on the page though

Reply:
Glad you tried it. For future reference the design system documentation is not the best. I don't think the people who write that are the devs.. I have found problems with it before. Best to check the component directly in the govuk frontend. but yeah it looks like it sets `govuk-fieldset__heading` in the isPageHeading code but that sounds like it's not a large font size based on your trials. 
https://github.com/alphagov/govuk-frontend/blob/main/packages/govuk-frontend/src/govuk/components/fieldset/template.njk